playwright/e2e/sample-reports-navigation.spec.ts

```diff
@@ -21,6 +21,21 @@ test.describe("Sample reports navigation", () => {
     test("URL contains secondaryReportType=status", async ({ page }) => {
       expect(page.url()).toContain("secondaryReportType=status");
     });
+
+    test("the page contains existing initiatives", async ({ page }) => {
+      const reportOnBtn = page.locator('select-issue-type').locator('button');
```

Comment:
Can you update your queries to not use CSS selectors and dom structure. They make the tests brittle since the structure or class name could change, the functionality still works, and the test fails. Additionally, since our users don't locate elements this way (by looking for dom nodes and CSS selectors) it lowers our confidence in the tests.

IIRC playwright implements a similar API to testing library:
- testing library: https://testing-library.com/docs/bs-react-testing-library/examples/#getbytext
- playwright: https://playwright.dev/docs/api/class-framelocator#frame-locator-get-by-text

Testing library has a good priority list for how to choose selectors based user's interacting with the test: https://testing-library.com/docs/queries/about/#priority

Reply:
Done!